### ISSUE TITLE
Add LA to lxc info CTNAME

### ISSUE THREAD

**Issue by s3rj1k** [lxc/lxd#6473]
Latest lxcfs supports reporting correct `/proc/loadavg`, would be nice to have load average returned by in lxc Info.

**Comment by stgraber** [comment]
I don't think we'll be doing that.

We have a very hard security requirement in LXD to never rely on information provided from the container's filesystem. Accessing `/proc/loadavg` right now can only be done by reading the file. That's different from the other metrics which we get by instead accessing the relevant cgroup metric from the host.

A container could mount a fuse filesystem on top of /proc and then cause the read to hang forever, potentially hanging LXD entirely.

I'd much prefer that once the cgroupv2 migration is done, we push for the inclusion of some kind of load average metric directly in the cgroupv2 hierarchy so we can safely retrieve this from the host, without the risk and cost associated with doing this in userspace.

@brauner think that'd be worth suggesting to relevant cgroup people? (addition of loadavg for a cgroup and its descendants).

**Comment by s3rj1k** [comment]
@stgraber I hope that after bringing cgroupv2 support, cgroupv1 code will stay for some time.

**Comment by brauner** [comment]
> I don't think we'll be doing that.
> 
> We have a very hard security requirement in LXD to never rely on information provided from the container's filesystem. Accessing `/proc/loadavg` right now can only be done by reading the file. That's different from the other metrics which we get by 

I'm a little confused here. The `/proc/loadavg` virtualization that LXCFS is doing is not accessing the containers filesystem at all afaik. It's calculating it via cgroups and falls back to the host `/proc/loadavg` in case calculating it from cgroups fails.
You can take a look at the original pr: https://github.com/lxc/lxcfs/pull/237
Maybe I missed something.

> @brauner think that'd be worth suggesting to relevant cgroup people? (addition of loadavg for a cgroup and its descendants).

The idea of doing it in-kernel is valid but extremely unlikely since Eric has nacked this some time ago already. The patch exists and has been done by the Siteground people. That's not a reason to not bring it up with the cgroup people again.

**Comment by stgraber** [comment]
@brauner for LXD to retrieve that value, we'd need to read /proc/loadavg from inside the container.

**Comment by brauner** [comment]
> @brauner for LXD to retrieve that value, we'd need to read /proc/loadavg from inside the container.

Oh, I overread the "returned from lxc info" part. Yeah...

**Comment by s3rj1k** [comment]
This would be nice to have at some point, especially then PSI Pressure Metrics were merged to cgroupv2
